test.php

```diff
@@ -2,107 +2,123 @@
 
 namespace App\Importer;
 
-use App\Entity\ClientImporter;
 use App\Entity\FieldConfiguration;
-use App\Model\ImportCriteria;
-use App\Model\SourceField;
 use DateTime;
 use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\DriverManager;
 use Doctrine\DBAL\Query\QueryBuilder;
-use Exception;
 use Google\Cloud\BigQuery\BigQueryClient;
 use MathParser\Exceptions\DivisionByZeroException;
 use MathParser\Interpreting\Evaluator;
-use MathParser\Parsing\Nodes\FunctionNode;
 use MathParser\StdMathParser;
 
 class BaseBigQueryImporter extends BaseImporter
```

Comment:
The `BaseImporter` class is not imported in this file to use like a abstract class.